### DISCUSSION TITLE
Strongswan example with inside and outside interface

### DISCUSSION THREAD

**Discussion by owensteam** [strongswan/strongswan#1183] [Q&A]
Hi there,

I'm used to having 2 interfaces on an VPN gateway - inside and outside. For the outside I usually use VRF-1 and for the inside VRF-2. the only way between the VRFs is via a tunnel (route based VPNs with virtual IP). 

Is there a comparable example available for Strongswan that anyone knows about (namespaces or similar)? I'm not sure how to get around having a default route on both the inside and outside interfaces.

Many thanks

**Comment by strongX509** [comment]
Have a look at route-based VTI or XFRM interfaces:

https://docs.strongswan.org/docs/5.9/features/routeBasedVpn.html

**Reply by owensteam** [reply]
I've had a good look today but I'd have to say that the examples are only really useful if you already know what you're doing. It doesn't look like you have an example of overlapping networks on inside and outside similar to how network OSs can do multitenant with VRF-lite.

I think maybe route tables might help? Is there a way of having the SA in one route table and the decrypted traffic in another route table?
